packages/css-framework/src/contexts/_light.scss

```diff
@@ -172,125 +177,125 @@
 
   // State Mapping
 
-  $states: (
-    "neutral": (
-      "default": color("neutral", "500"),
-      "faded": color("neutral", "200"),
-    ),
-    "action": (
-      "default": color("action", "500"),
-      "faded": color("action", "200"),
-    ),
-    "danger": (
-      "default": color("danger", "500"),
-      "faded": color("danger", "200"),
-    ),
-    "warning": (
-      "default": color("warning", "500"),
-      "faded": color("warning", "200"),
-    ),
-    "success": (
-      "default": color("success", "500"),
-      "faded": color("success", "200"),
-    ),
-  );
+  // defaults.$states: (
```

Comment:
Does this stuff need to stay in? Can we just remove it if it's commented out?

Reply:
Yeah good shout, I'll remove it.